config/samples/metal3.io_v1alpha1_hostfirmwaresettings.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: metal3.io/v1alpha1
+kind: HostFirmwareSettings
+metadata:
+  name: hostfirmwaresettings-sample
+spec:
+  # Add fields here
+  foo: bar
```

Comment:
Looks like this needs to be regenerated :)

Reply:
Yeah I made need to patch this after regeneration, will fix.